src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/Service/overview.html.twig

```diff
@@ -4,7 +4,85 @@
+                        <th>{{ 'service.overview.entitylist.entityId'|trans }}</th>
+                        <th>{{ 'service.overview.entitylist.protocol'|trans }}</th>
+                        <th>{{ 'service.overview.entitylist.state'|trans }}</th>
+                    </tr>
+                    {% set hasEntities = false %}
+                    {% for entity in service.entityList %}
+                        {% if entity.environment == 'production' %}
+                            {% set hasEntities = true %}
+                            <tr>
+                                <td><a href="{{ entity.link }}">{{ entity.name }}</a></td>
+                                <td><a href="{{ entity.link }}">{{ entity.entityId }}</a></td>
+                                <td><a href="{{ entity.link }}">{{ entity.protocol }}</a></td>
+                                <td><a href="{{ entity.link }}">{{ entity.state }}</a></td>
+                            </tr>
+                        {%  endif %}
+                    {% endfor %}
+                    {% if hasEntities == false %}
+                        <tr>
+                            <td colspan="4">{{ 'service.overview.entitylist.empty'|trans }}</td>
+                        </tr>
+                    {% endif %}
+
+                    {# Spacing #}
+                    <tr>
+                        <td colspan="4"></td>
+                    </tr>
+
+                    {# Test entities #}
+                    <tr>
+                        <td colspan="4">
+                            <h3>{{ 'service.overview.entitylist.test.title'|trans }}</h3>
+                        </td>
+                    </tr>
+                    <tr>
+                        <th>{{ 'service.overview.entitylist.name'|trans }}</th>
+                        <th>{{ 'service.overview.entitylist.entityId'|trans }}</th>
+                        <th>{{ 'service.overview.entitylist.protocol'|trans }}</th>
+                        <th>{{ 'service.overview.entitylist.state'|trans }}</th>
+                    </tr>
+                    {% set hasEntities = false %}
+                    {% for entity in service.entityList %}
+                        {% if entity.environment == 'test' %}
+                            {% set hasEntities = true %}
+                            <tr>
+                                <td><a href="{{ entity.link }}">{{ entity.name }}</a></td>
+                                <td><a href="{{ entity.link }}">{{ entity.entityId }}</a></td>
+                                <td><a href="{{ entity.link }}">{{ entity.protocol }}</a></td>
+                                <td><a href="{{ entity.link }}">{{ entity.state }}</a></td>
+                            </tr>
+                        {%  endif %}
+                    {% endfor %}
+                    {% if hasEntities == false %}
+                        <tr>
+                            <td colspan="4">{{ 'service.overview.entitylist.empty'|trans }}</td>
```

Comment:
Did you consider removing the table entirely if no entities are present? Or is that not in accordance with @teunfransen s design?

Reply:
There was nothing in the design, and this was the most easy thing to do at the moment.